app/controllers/reviews_controller.rb

```diff
@@ -36,6 +36,13 @@ def update
         end
     end
 
+    def destroy
+      @shelter = Shelter.find(params[:shelter_id])
+      @review = Review.find(params[:review_id])
```

Comment:
Covered in person but need to find relevant review AND shelter before we can destroy